pkgs/development/guile-modules/guile-ssh/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, guile, libssh
+, autoreconfHook, pkg-config, texinfo, which
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guile-ssh";
+  version = "0.12.0";
+
+  src = fetchFromGitHub {
+    owner = "artyom-poptsov";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "054hd9rzfhb48gc1hw3rphhp0cnnd4bs5qmidy5ygsyvy9ravlad";
+  };
+
+  configureFlags = [ "--with-guilesitedir=\${out}/share/guile/site" ];
+
+  postFixup = ''
+    for f in $out/share/guile/site/ssh/**.scm; do \
```

Comment:
Its somewhat hard to believe that something used in `guix` is not using fully-qualified paths. Why is this necessary?

Reply:
See https://github.com/guix-mirror/guix/blob/35c43fcdbb408a5755efebc13241dd9082360aa1/gnu/packages/ssh.scm#L275-L277